spec/services/measurements_creator_spec.rb

```diff
@@ -0,0 +1,148 @@
+require "spec_helper"
+
+describe MeasurementsCreator do
+  describe "#self.call" do
```

Comment:
In Ruby lingo you use a `.` for class methods and `#` for instance methods. So you could go with `.call` here